.github/workflows/e2e_tests.yaml

```
@@ -98,6 +98,7 @@ jobs:
           script: |
             sleep 15
             flutter test integration_test/e2e_test.dart --dart-define=CI=true
+            $ANDROID_HOME/platform-tools/adb logcat '*:E' > adb-log.txt &
```

```suggestion
            $ANDROID_HOME/platform-tools/adb logcat '*:E' > adb-log.txt &
            flutter test integration_test/e2e_test.dart --dart-define=CI=true
```

I think this makes more sense, right?